Components/Helper/Price.php

```diff
@@ -234,8 +236,6 @@ private static function getProductPriceRateAfterDiscount(Article $article, \Shop
      */
     public static function format($price)
     {
-        /** @var NostoHelperPrice $helper */
-        $helper = Nosto::helper('price');
-        return $helper->format($price);
+        return NostoPriceHelper::format($price);
```

Comment:
We can unwrap this.

Reply:
I'll deprecate and use the root format function directly